07_cmd_lab/cmdlab.txt

```diff
@@ -3,11 +3,36 @@ Questions online: https://www.tocode.co.il/bundles/unix1/lessons/cmdlab
 
 1. Login to your UNIX account. How do you find:
   - your current shell?
+ $ echo $0
+ bash
+  
   - your default shell?
+  bash
+  
   - a list of all shells currently installed on the machine?
+  $ cat /etc/shells
+  # /etc/shells: valid login shells
+  /bin/sh
+  /bin/dash
+  /bin/bash
+  /bin/rbash
+  
 
 2. View the file /etc/passwd on your machine with the command "cat /etc/passwd". Then read about the file in "man 5 passwd". What information is held in the file? What are the fields in each line?
+/etc/passwd contains one line for each user account. The fields are: 
+- login name
+- optional encrypted password
+- numerical user ID
+- numerical group ID
+- user name or comment field
+- user home directory
+- optional user command interpreter
+
 
 3. In Unix, the /etc folder holds configuration files. Display all files in "/etc" folder ordered by file size (Hint: search in "man ls")
+ls -ltr /etc
```

Comment:
hmm...  ```ls -ltr ```  does not work... let's check the man page.. 
type ``` man ls ``` and find out what are the flags "l" "t" and "r"

i'll start:

-r reverse  order while sorting.
-l ?????
-t  ?????

now it's your turn :)